Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -2077,7 +2077,7 @@ WebsiteDataStoreParameters WebsiteDataStore::parameters()
         SandboxExtension::createHandleForReadWriteDirectory(parameters.serviceWorkerRegistrationDirectory, parameters.serviceWorkerRegistrationDirectoryExtensionHandle);
 #endif
 
-    platformSetNetworkParameters(parameters);
+    //platformSetNetworkParameters(parameters);
```

Comment:
Please do not edit generic WebKit headers, unless they are out of sync with upstream webkit code because of an earlier merge problem.

Reply:
@pulkomandy i was trying to debug thats why commented it . iforgot to revert it back 😕 sorry.